pkg/types/openstack/validation/machinepool.go

```diff
@@ -20,5 +21,35 @@ func ValidateMachinePool(p *openstack.MachinePool, fldPath *field.Path) field.Er
 		}
 	}
 
+	allErrs = append(allErrs, validateUUIDV4s(p.AdditionalNetworkIDs, fldPath.Child("additionalNetworkIDs"))...)
```

Comment:
Should we be using the valid values fetcher here to validate that these networks and security groups actually exist in openstack?

Reply:
This will be done as a part of our Validation epic